code/parse/sexp.cpp

```diff
@@ -11981,33 +11982,43 @@ void sexp_cap_waypoint_speed(int n)
 
 /**
  * Causes a ship to jettison its cargo
- * note that the 2nd arg (jettison delay) is not implemented
  */
-void sexp_jettison_cargo(int n)
+void sexp_jettison_cargo(int n, bool jettison_new)
 {
 	char *shipname;
 	int ship_index;
+	int jettison_delay_or_speed;
```

Comment:
But this function is used for `jettison-cargo-delay` as well as the new sexp.  And in that context, the value is an integral number of seconds.  (Note the name of the variable.)  You'd be casting an int to a float and then back to an int again.